### DISCUSSION TITLE
dropdown dropdownMenue Link error

### DISCUSSION THREAD

**Discussion by Sofiane-Bahmed** [heroui-inc/heroui#2163] [Help]
i've implemented a Link inside dropdowItem to navigate to a specific route, but this error message appears : Unhandled Runtime Error
Error: Unknown element <[object Object]> in collection. 
Help i didnt fiind a solution ?!!!!!

**Comment by soxamz** [comment]
## Try adding `"use client"` in the first line of your code. I guess it will solve the issue.

**Comment by Sofiane-Bahmed** [comment]
> ## Try adding `"use client"` in the first line of your code. I guess it will solve the issue.
i have the "use client" but the error appears

**Comment by soxamz** [comment]
### Hmm, I guess I can suggest you a better way. This is as following:-
-  Remove the Link tag inside the DropdownItem tag and instead just write the text which you want to display as the dropdown item. 

- Add the href attribute in the opening DropdownItem tag and mention the respective path inside it, for example like this : `<DropdownItem href="/about">About</DropdownItem>`

- And at last but not the least, don't forget to mention `"use client"` at the very top of the code.

If everything goes well, your code will work fine. _Hope it helps._

**Reply by Sofiane-Bahmed** [reply]
the solution was the href attribut inside the dropdownItem tag, i fouond it in the documentation of nextUI (Routing), so your suggesting is right, thank you brotha for your answers

**Reply by soxamz** [reply]
No problem! 👍
